src/libraries/System.Data.SqlClient/tests/StressTests/System.Data.SqlClient.Stress.Tests/System.Data.SqlClient.Stress.Tests.csproj

```diff
@@ -3,8 +3,9 @@
     <RootNamespace>Stress.Data.SqlClient</RootNamespace>
     <AssemblyName>System.Data.SqlClient.Stress.Tests</AssemblyName>
     <!-- These should not run in Helix: they produce no outputs and are only run manually -->
+    <ArchiveTest>false</ArchiveTest>
```

Comment:
I believe we can remove this property, we now use `IgnoreForCI`, right, @ViktorHofer ?

Reply:
Weird that I didn't capture this case with my change. Yes please use [IgnoreForCI](https://github.com/dotnet/runtime/blob/master/eng/testing/tests.targets#L23)